ddprof-lib/src/main/cpp/profiler.cpp

```diff
@@ -355,6 +355,12 @@ int Profiler::getJavaTraceAsync(void* ucontext, ASGCT_CallFrame* frames, int max
         return 0;
     }
 
+    if (VM::isHotspot() && vm_thread->lastJavaSP() == 0 && vm_thread->lastJavaPC() != 0) {
```

Comment:
This is just a nit - would it make sense to do `vm_thread->lastJavaSP() == NULL` since the comment is talking about NULL value (I know they are the same thing, that's why this is just a nit)